src/vanilla/harmonium.js

```diff
@@ -285,4 +285,5 @@ export function initializeAllComponents() {
   initializeDrawerComponents()
   initializeSliderComponents()
   initializeExpandingColumnComponents()
+  initializeStickyComponents()
```

Comment:
@bryanjos I've got an issue for you with this 👶:
https://github.com/revelrylabs/harmonium/issues/477

Reply:
Thanks!